lib/form/scripts.js

```diff
@@ -24,6 +24,17 @@ const rendered = (type) => {
     let plugins = type.attributes.map(plugin => {
       return plugin.type;
     });
+
+    // Checks if plugin directory has values
+    if (!Array.isArray(dir)) {
+      rej(new Error('Plugin directory congiguration must be an array'));
+    }
+
+    // add node_modules if it's not in there
+    if (dir.indexOf(path.join(__dirname, '../../node_modules')) === -1) {
```

Comment:
Please, write an issue, we'll discuss, then a PR. Don't try and just throw new stuff in to an existing PR